services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/application/ApplicationArea.kt

```diff
@@ -10,5 +12,19 @@ sealed interface ApplicationArea {
 data class CableReportApplicationArea(override val name: String, override val geometry: Polygon) :
     ApplicationArea
 
-data class ExcavationNotificationArea(override val name: String, override val geometry: Polygon) :
-    ApplicationArea
+data class ExcavationNotificationArea(
+    override val name: String,
+    override val geometry: Polygon,
+    val hankealueId: Int,
+    val tyoalueet: List<Tyoalue>,
+    val katuosoite: String,
+    val tyonTarkoitukset: Set<TyomaaTyyppi>,
+    val lisatiedot: String,
+    val haitat: Haitat,
```

Comment:
On the ticket this was specified as
```kotlin
    meluhaitta: Meluhaitta
    polyhaitta: Polyhaitta
    tarinahaitta: Tarinahaitta
    kaistahaitta: VaikutusAutoliikenteenKaistamaariin
    kaistahaittojenPituus: AutoliikenteenKaistavaikutustenPituus
```
Not this is trying to implement the Haittojenhallintasuunnitelma?

Reply:
I got carried away again when I tried to follow the ux design. Reverted to contain only those mentioned fields.